src/index.html

```diff
@@ -5,10 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img class="logo"
+        src="./images/logo.png"
+        alt="logo_moyo"
+        >
```

Comment:
Check `checklist`
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md
```suggestion
        <img 
          class="logo"
          src="./images/logo.png"
          alt="logo_moyo"
        >
```